src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -10,28 +10,29 @@
 
 from sqlfluff.core.dialects import load_raw_dialect
 from sqlfluff.core.parser import (
+    AnyNumberOf,
     Anything,
     BaseSegment,
-    OneOf,
-    Ref,
-    Sequence,
     Bracketed,
+    CodeSegment,
+    Dedent,
     Delimited,
-    AnyNumberOf,
+    Indent,
     KeywordSegment,
-    SymbolSegment,
-    RegexLexer,
-    StringLexer,
-    CodeSegment,
+    Matchable,
     NamedParser,
-    StringParser,
-    RegexParser,
     Nothing,
-    StartsWith,
+    OneOf,
     OptionallyBracketed,
-    Indent,
-    Dedent,
-    Matchable,
+    Ref,
+    RegexLexer,
+    RegexParser,
+    SegmentGenerator,
+    Sequence,
+    StartsWith,
+    StringLexer,
+    StringParser,
+    SymbolSegment,
```

Comment:
Side note: That's quite a list of imports! 😅

Reply:
Ordered alphabetically so looks bigger than it is. Think I only added `SegmentGenerator` and `RegexParser` in this PR.

Reply:
Oh yeah, I didn't mean anything was wrong. It's just rare in Python to see an import list that long. If I start "importing the world", sometimes I switch to something like:
```
from sqlfluff.core import parser
```

But that requires all the references in that module to be prefixed with `parser.`.

Just ignore me. I'm feeling talkative on a late winter morning...

Reply:
Shorter than ANSI (just!).